.gitignore

```diff
@@ -3,4 +3,6 @@
 /caddy/init/Caddyfile
 /config/vnc_rsa*
 /config/passwords
-/config/caddy_passwords
\ No newline at end of file
+/config/caddy_passwords
+**/.env
```

Comment:
The .env files are staged, so this will not work unfortunately. Maybe I'll find a better solution here later, but could you remove this line for now? The pycache is fine.

Reply:
I've removed the .env file from GIT, that's more clean now.

Reply:
Understood. Done. Thank you.

Reply:
And sorry for the confusion...